docs/source/deployment/deploy-model-to-kubernetes/index.rst

```diff
@@ -1,476 +1,46 @@
-Deploying a model to Kubernetes
-===============================
-
-This guide showcases how you can use MLflow end-to-end to:
-
-- Train a linear regression model
-- Package the code that trains the model in a reusable and reproducible model format
-- Deploy the model into a simple HTTP server that will enable you to score predictions
-
-**Note** The modeling usecase shown here utilizes a common dataset to predict the quality of wine based on quantitative features
-like the wine's "fixed acidity", "pH", "residual sugar", and so on. The dataset
-is from UCI's `machine learning repository <http://archive.ics.uci.edu/ml/datasets/Wine+Quality>`_.
-[1]_
-
-.. contents:: Table of Contents
-  :local:
-  :depth: 1
-
-What You'll Need
+Deploy MLflow Model to Kubernetes
+=================================
+Using MLServer as the Inference Server
+--------------------------------------
+By default, MLflow deployment uses `Flask <https://flask.palletsprojects.com/en/1.1.x/>`_, a widely used WSGI web application framework for Python,
```

Comment:
It might be nice to provide either a link or a brief description of what WSGI is (it can help to inform why we're saying this about Flask :) ) since the reading audience might not be familiar with the differences between a standard Web Server Gateway Interface and something else (like ASGI or the built-in optimizations for inference serving that MLServer from Seldon has). 
What do you think about some brief education for readers on these topics so that they know why we're talking about and supporting MLServer on k8s?

Reply:
The comparison is briefly described here, tho I don't add low-level details on why: https://output.circle-artifacts.com/output/job/0576e156-79b5-48d6-a81d-8ceafb03de8b/artifacts/0/docs/build/html/deployment/deploy-model-locally.html#serving-frameworks

I will write a bit more specific details there and put link to it here.

Reply:
Perfect! :D